core/README.md

```diff
@@ -125,18 +125,19 @@ DataYoga supports both SQL and [JMESPath](https://jmespath.org/) expressions. JM
+| Function             | Description                                                                                                                          | Example                                                                                                                                           | Comments                                                                                                                                                         |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `capitalize`         | Capitalizes all the words in the string                                                                                              | Input: `{"name": "john doe"}` <br /> Expression: `capitalize(name)` <br /> Output: `John Doe`                                                     |                                                                                                                                                                  |
+| `concat`             | Concatenates an array of variables or literals                                                                                       | Input: `{"fname": "john", "lname": "doe"}` <br /> Expression: `concat([fname, ' ' ,lname])` <br /> Output: `john doe`                             | This is equivalent to the more verbose built-in expression: `' '.join([fname,lname])`                                                                            |
+| `hash`               | Calculates a hash using the `hash_name` hash function and returns its hexadecimal representation                                     | Input: `{"some_str": "some_value"}` <br /> Expression: `hash(some_str, &#96;sha1&#96;)` <br /> Output: `8c818171573b03feeae08b0b4ffeb6999e3afc05` | Supported algorithms: sha1 (default), sha256, md5, sha384, sha3_384, blake2b, sha512, sha3_224, sha224, sha3_256, sha3_512, blake2s                              |
+| `in`                 | Checks if the iterable contains the given element                                                                                    | Input: `{"el": "b"}` <br /> Expression: ``in(el, `["a", "b", "c"]`)`` <br /> Output: `True`                                                       |                                                                                                                                                                  |
```

Comment:
Change description to:
Checks if an element matches any value in a list of values

Reply:
✅